pages/src/App.js

```diff
@@ -1,6 +1,6 @@
+//import io from 'socket.io';
 import React, { Component } from 'react'
 import { Map } from 'immutable'
-
```

Comment:
Why did you remove this line?

Reply:
It was just an empty line, i don't remembre why.

Reply:
Try to respect the coding style, indentation, etc. :)